manifests/tftp.pp

```diff
@@ -13,8 +13,7 @@
     recurse => true;
   }
 
-  foreman_proxy::tftp::sync_file{$foreman_proxy::tftp_syslinux_files:
```

Comment:
Maybe put the new block in an "else" clause of `if $foreman_proxy::tftp_syslinux_files`?